sqlmesh/core/linter/rule.py

```diff
@@ -22,6 +22,20 @@ class RuleLocation(PydanticModel):
     start_line: t.Optional[int] = None
 
 
+class Position(PydanticModel):
+    """The position of a rule violation in a file, the position follows the LSP standard."""
+
+    line: int
+    character: int
```

Comment:
Does "character" mean column offset in the line? Why not use `column` or something similar so that the naming is consistent?

I also suggest we start documenting some details, like whether it's 1-based indexing and stuff, so reading the docstring makes it obvious to understand what's going on.